src/main/java/de/sakpaas/backend/exception/ErrorAttributes.java

```diff
@@ -0,0 +1,148 @@
+package de.sakpaas.backend.exception;
+
+import java.nio.file.AccessDeniedException;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.web.servlet.error.DefaultErrorAttributes;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.request.WebRequest;
+
+/**
+ * Used to react to any kind of error that happens during the launch of the application.
+ */
+@Component
+public class ErrorAttributes extends DefaultErrorAttributes {
+
+  // Object Key in the Response for the Frontend
+  private static final String RESPONSE_OBJECT_KEY = "AppError";
+
+  // Error types that are needed for specific reaction at the frontend
+  private static final String DEFAULT_ERROR_TYPE = "error";
+  private static final String UNKNOWN_ERROR_TYPE = "unknown";
+  private static final String INTERNAL_ERROR_TYPE = "internal";
+  private static final String NO_PERMISSION_ERROR_TYPE = "permission";
+  private static final String UNKNOWN_RESOURCE_ERROR_TYPE = "resource";
+  private static final String NOT_AUTHENTICATED_ERROR_TYPE = "authentication";
+
+
+  // TextIds that are used inside this class
+  private static final String UNKNOWN_ERROR_ID = "unknown_error";
+  private static final String INTERNAL_ERROR_ID = "internal_error";
+  private static final String NO_PERMISSION_ERROR_ID = "no_permission";
+  private static final String UNKNOWN_RESOURCE_ID = "unknown_resource";
+  private static final String NOT_AUTHENTICATED_ID = "not_authenticated";
+
+  @Value("${app.exceptions.debug}")
+  private boolean printException;
+
+
+  /**
+   * The String that is used as the prefix for the replacers.
+   */
+  private static final String REPLACER = "%%";
+
+  /**
+   * Returns the message that corresponds to the given textId.
+   * 
+   * @param textId the ID of the message that shall be received
+   * @return the corresponding message
+   */
+  private String getErrorMessage(String textId) {
+    // TODO Should be implemented in any kind of text management as soon as it exists

```

Comment:
Hier ist noch ein TODO

Reply:
Da die Applikation noch kein Textmanagement besitzt war ich nicht in der Lage das einzubauen. Sobald es ein Textmanagement gibt muss das dort (bestenfalls mehrsprachig) implementiert werden. Sollte definitiv in den nächsten Sprint mit aufgenommen werden